src/paragraphcommand.js

```diff
@@ -69,10 +72,28 @@ export default class ParagraphCommand extends Command {
 	 * @private
 	 */
 	_updateValue() {
```

Comment:
In list command and block quote command this is a public method called `refreshValue()`. We can align paragraph command to them.